resources/views/prueba/index.blade.php

```diff
@@ -0,0 +1 @@
+@extends('layouts.main')
```

Comment:
Maybe its not a good idea to commit testing files. 
If you want you can do it thought as we are not in release yet.